src/main/java/nl/tudelft/pixelperfect/game/Game.java

```diff
@@ -175,8 +176,10 @@ public void onAction(String name, boolean keyPressed, float tpf) {
           rotateRight = keyPressed;
         } else if (name.equals("start")) {
           startKey = keyPressed;
-        } else if (name.equals("debug")) {
-          debugKey = keyPressed;
+        } else if (name.equals("debugOn")) {
+          debugKeyOn = keyPressed;
+        } else if (name.equals("debugOff")) {
+          debugKeyOff = keyPressed;
         }
```

Comment:
This method should be separated into a key pressed part (movement) and on key press  (debug + start). This is probably quite easy to fix.